Checkpoint1/Program5/Program.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections;
+
+namespace program5
+{
+    class Program
+    {
+            public static void Main()
+            {
+
+                // set sort to false for initialization purposes
+                bool sort = false;
+                // prompt user to enter a series of numbers separated by commas
+                Console.WriteLine("Please enter a series of numbers separated by commas");
+                // read the user input into a string variable
+                string numbers = Console.ReadLine();
+                // separate the string into an array based on commas between numbers
+                string[] strNumbersArr = numbers.Split(',');
+                // initialize an integer array with the same length as the previous string array
+                int[] intNumbersArr = new int[strNumbersArr.Length];
+                // display first part of a message that shows user numbers they entered
+                Console.WriteLine("The numbers you entered were: ");
+
+            // loop through string array and assign the intNumbersArr index the int value of strNumberArr index
+            for (int i = 0; i < strNumbersArr.Length; i++)
+                {
+                    intNumbersArr[i] = int.Parse(strNumbersArr[i]);
+                }
+            
+            // output each index of the intNumbersArr
+            for (int i = 0; i < intNumbersArr.Length; i++)
+            {
+                Console.WriteLine(intNumbersArr[i]);
+            }
+
+            // pass in intNumbersArr to findGreatest function and assign result to greatestNum
+            int greatestNum = findGreatest(intNumbersArr, sort);
+            
+            // display message showing the greatest number of numbers that user entered
+            Console.WriteLine("The greatest number out of the numbers you entered is: {0}", greatestNum);
+            
+                
+                       
+        }
+
+        // function to find the greatest number in an array of numbers
+        public static int findGreatest(int[] intNumbersArr, bool sort)
+            {
+                 // loop through array
+                        for (int i = 0; i < intNumbersArr.Length - 1; i++)
```

Comment:
You don't need to sort in order to determine greatest. See if you can find the greatest without looping more than once.